docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
```

Comment:
I do not see any configuration to make debugging of the application available. Does debugging work?